services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events.py

```diff
@@ -443,14 +442,13 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
             or scheduler_data.dynamic_sidecar.swarm_network_name is None
             or scheduler_data.proxy_admin_api_port is None
         ):
-            raise ValueError(
-                "Did not expect None for any of the following: "
-                f"{scheduler_data.dynamic_sidecar.dynamic_sidecar_id=} "
-                f"{scheduler_data.dynamic_sidecar.dynamic_sidecar_network_id=} "
-                f"{scheduler_data.dynamic_sidecar.swarm_network_id=} "
-                f"{scheduler_data.dynamic_sidecar.swarm_network_name=} "
-                f"{scheduler_data.proxy_admin_api_port=}"
+            msg = (
```

Comment:
I'm not sure why, but this is less readable now.

Reply:
actually I do agree. will change it. nevertheless the comment is a bit weird...